Dockerfile

```diff
@@ -71,4 +71,4 @@ RUN mkdir /bzt-configs \
   && rm -rf /tmp/*
 
 WORKDIR /bzt-configs
-CMD bzt -l /tmp/artifacts/bzt.log /bzt-configs/*.yml
+ENTRYPOINT ["sh", "-c", "bzt /bzt-configs/*.yml -l /tmp/artifacts/bzt.log \"$@\"", "ignored"]
```

Comment:
Ah, so you are thinking that the user should just pass the config files in as additional `docker run` command line arguments? Something like this:

    docker run undera/taurus myConfig1.yml myConfig2.yml

That would certainly simplify the Docker file, and it would also allow using .json config files. It is worth noting though that configuring the ENTRYPOINT like that would definitely be a breaking change, and would need a new major version if you're following SemVer. I'll go ahead and make that change to this PR and then you can decide if it's worth keeping.

I'm glad I can help!